File: external-mock/apps/apimock/src/main/java/com/bmw/fd/apimock/Application.java
package com.bmw.fd.apimock;

import com.bmw.fd.spring.api.doc.ApidocConfiguration;
import com.bmw.fd.spring.api.error.ApiExceptionHandler;
import com.bmw.fd.spring.api.security.JwtConfig;
import com.bmw.fd.spring.configuration.EnableSpringBase;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.context.ApplicationContext;
import org.springframework.context.annotation.Bean;
import org.springframework.context.annotation.Import;

import java.util.UUID;


@SpringBootApplication
//@EnableSpringBase
@Import({ApidocConfiguration.class, JwtConfig.class})
public class Application {

    public static void main(String[] args) {
        ApplicationContext ctx = SpringApplication.run(Application.class, args);
    }

    @Bean
    public ApiExceptionHandler springServiceExceptionHandler() {
        return new ApiExceptionHandler();
    }
}

File: src/main/kotlin/de/unia/se/teamcq/Application.kt
package de.unia.se.teamcq

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.cache.annotation.EnableCaching

@EnableCaching
@SpringBootApplication
class Application

fun main(args: Array<String>) {
    SpringApplication.run(Application::class.java, *args)
}
